sgkit/stats/aggregation.py

```diff
@@ -285,6 +285,143 @@ def call_rate(ds: Dataset, dim: Dimension, call_genotype_mask: Hashable) -> Data
     )
 
 
+@numba_jit
+def _greatest_common_denominator(x: int, y: int) -> int:
+    while y != 0:
+        t = x % y
+        x = y
+        y = t
+    return x
+
+
+# TODO: replace with math.comb when supported by numba
+@numba_jit
+def _comb(n: int, k: int) -> int:
+    if k > n:
+        return 0  # pragma: no cover
+    r = 1
+    for d in range(1, k + 1):
+        gcd = _greatest_common_denominator(r, d)
+        r //= gcd
+        r *= n
+        r //= d // gcd
+        n -= 1
+    return r
+
+
+@numba_jit
```

Comment:
It would be good to have a handful of standalone tests for these functions to check some boundary cases against some other implementation (or just known results).